src/components/auction/OrderPlacement/index.tsx

```diff
@@ -326,18 +341,33 @@ const OrderPlacement: React.FC<OrderPlacementProps> = (props) => {
     !!account &&
     !!biddingToken.address
 
+  const auctioningTokenAddress = auctioningToken && auctioningToken?.address
+  const linkForKYC = auctioningTokenAddress ? kycLinks[auctioningTokenAddress] : null
   return (
     <>
       <Wrapper>
         {auctionInfoLoading && <InlineLoading size={SpinnerSize.small} />}
-        {!auctionInfoLoading && isPrivate && !signatureAvailable && (
+        {!auctionInfoLoading && isPrivate && !signatureAvailable && account == null && (
```

Comment:
it may also be confusing just to show the button `Connect Wallet`, how about changing the `account` logic to show that part?

            <PrivateWrapper>
              <LockBig />
              <TextBig>Private auction</TextBig>
              <EmptyContentTextNoMargin>
                You need to get allowed to participate.
              </EmptyContentTextNoMargin>
            </PrivateWrapper>
            {account == null ? (
              <ActionButton onClick={toggleWalletModal}>Connect Wallet</ActionButton>
            ) : (
              <EmptyContentTextSmall>
                {linkForKYC ? (
                  <ExternalLink href={linkForKYC}>Get Allowed ↗</ExternalLink>
                ) : (
                  <EmptyContentTextSmall>
                    Ask the auctioneer to get allow-listed.
                  </EmptyContentTextSmall>
                )}{' '}
              </EmptyContentTextSmall>
            )}


![Selection_320](https://user-images.githubusercontent.com/4270166/126666959-f8ed7c84-f599-4f96-848b-7d3f1bb13cdd.png)

Reply:
Nice, I like this one. Leet's do it